k8s/errors/cronjob-late-maybe-error.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cronjob-late-maybe-error
+  labels:
+    type: test-pod
+spec:
+  schedule: "* * * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            type: test-pod
+            run: cronjob-late-maybe-error
+        spec:
+          containers:
+            - name: hello
+              image: busybox:1.28
+              imagePullPolicy: IfNotPresent
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  MINWAIT=0
+                  MAXWAIT=60
+                  sleep $((MINWAIT+RANDOM % (MAXWAIT-MINWAIT)))
+                  sleep 3
+                  r=$((RANDOM%2))
+                  if [ $r -eq 0 ]; then echo Hello!; fi
+                  if [ $r -eq 1 ]; then exit 1; fi
```

Comment:
Here and in the other error file: let's use `else`, now it looks a bit awkward as if we're unsure if we only get 0 or 1 back.